source/administration/mmctl-cli-tool.rst

```diff
@@ -0,0 +1,2320 @@
+mmctl Command Line Tool (Beta)
+==============================
+
+The mmctl tool is a remote CLI tool for Mattermost which is installed locally and uses the Mattermost API. Authentication
+is done with either login credentials or an authentication token. <elaborate more on this, provide links to other docs>.
```

Comment:
There is an internal note here to elaborate / add links

Reply:
Thanks for catching that!